tarpc_examples/src/bin/throughput.rs

```diff
@@ -41,14 +38,16 @@ fn bench_tarpc(target: u64) {
         nread += CHUNK_SIZE as u64;
     }
     let duration = time::Instant::now() - start;
-    println!("TARPC: {}MB/s",  (target / (1024 * 1024)) as u64 / duration.as_secs());
+    println!("TCP:   {}MB/s",
```

Comment:
I think you mean TARPC here? Please take some care to make sure the `{}MB/s` lines up between TARPC and TCP :)